src/XmlDocMarkdown.Core/MarkdownGenerator.cs

```diff
@@ -431,6 +431,7 @@ private NamedText WriteMemberPage(string path, IReadOnlyList<MemberInfo> memberG
 						seeAlsoMembers.Add(declaringType);
 
 					foreach (var seeAlso in seeAlsoMembers
+						.Where(x => x != memberInfo)
```

Comment:
I don't know how confident I was in its necessity, but I usually compare members via `XmlDocUtility.GetXmlDocRef` rather than by reference equality.